### PR TITLE
Bug 1883837, updated Managing Weights procedure

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1883837

Preview build: https://routes-managing-weights--ocpdocs.netlify.app/openshift-enterprise/latest/applications/deployments/route-based-deployment-strategies.html#deployments-ab-testing-lb-web_route-based-deployment-strategies

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
lgtm

**Comment by ahardin-rh** [comment]
@yapei Are you the correct contact to provide QE review? Thank you!

File: modules/deployments-ab-testing-lb.adoc [hunk 1]
```diff
@@ -113,23 +113,36 @@ spec:
...
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking -> *Routes* details page.
 
-. Select *Edit* from the Actions menu.
+. Click the Options menu {kebab} next to the Route you want to edit and select *Edit Route*.
```
Comment (yapei) [inline modules/deployments-ab-testing-lb.adoc:122]: If a user is on Route details page, no Options menu, there is only Actions menu, see below screenshot
Screen Shot 2020-11-05 at 9 49 33 AM

If a user is on Routes list page, he can see Options menu, see below screenshot
Screen Shot 2020-11-05 at 9 51 02 AM

Comment (yapei) [reply modules/deployments-ab-testing-lb.adoc:122]: Also it seems an extra * in preview build
Screen Shot 2020-11-05 at 9 53 45 AM

File: modules/deployments-ab-testing-lb.adoc [hunk 2]
```diff
@@ -113,23 +113,36 @@ spec:
...
+
+. Click the *Create Route* button.
+
+. Enter the route's *Name*.
+
+. Select the *Service*.
+
+. Click *Add Alternate Service*.
+
+. Enter a value for *Weight*. Enter a number between `0` and `255` that depicts relative weight compared with other targets. The default is `100`.
```
Comment (yapei) [inline modules/deployments-ab-testing-lb.adoc:141]: IMO here we should mention `Enter values for *Weight* and *Alternate Service Weight*`

File: modules/deployments-ab-testing-lb.adoc [hunk 3]
```diff
@@ -113,23 +113,36 @@ spec:
...
-specified by integers between 0 and 256 for each service.
-+
-Traffic weightings are shown on the *Overview* in the expanded rows of the
-applications between which traffic is split.
+. Click *Save*.
+
+[id="deployments-ab-testing-lb-web-new-route_{context}"]
+== Managing weights of an new route using the web console
+
+. Navigate to the *Networking -> *Routes* details page.
```
Comment (yapei) [inline modules/deployments-ab-testing-lb.adoc:131]: it should be `Navigate to the *Networking -> *Routes* list page`

File: modules/deployments-ab-testing-lb.adoc [hunk 4]
```diff
@@ -113,23 +113,36 @@ spec:
...
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking* -> *Routes* page.
 
-. Select *Edit* from the Actions menu.
+. Click the Actions menu {kebab} next to the Route you want to edit and select *Edit Route*.
 
-. Check *Split traffic across multiple services*.
+. Edit the YAML file. Update the `weight` to be an integer between `0` and `256` that specifies the  target's relative weight against other target reference objects. The value `0` suppresses requests to this back-end. The default is `100`. Run `oc explain routes.spec.alternateBackends` for more information about the options.
```
Comment (bobfuru) [inline modules/deployments-ab-testing-lb.adoc:124]: s/back-end/back end because it is a noun here

File: modules/deployments-ab-testing-lb.adoc [hunk 5]
```diff
@@ -113,23 +113,36 @@ spec:
...
+. Click *Save*.
+
+[id="deployments-ab-testing-lb-web-new-route_{context}"]
+== Managing weights of an new route using the web console
+
+. Navigate to the *Networking* -> *Routes* page.
+
+. Click the *Create Route* button.
+
+. Enter the route's *Name*.
```
Comment (bobfuru) [inline modules/deployments-ab-testing-lb.adoc:135]: s/route's/route

File: modules/deployments-ab-testing-lb.adoc (hunk 4 above)
Comment (bobfuru) [reply modules/deployments-ab-testing-lb.adoc:124]: s/target's relative weight/relative weight of the target/ (ISG suggests avoiding contractions)

File: modules/deployments-ab-testing-lb.adoc [hunk 6]
```diff
@@ -113,23 +113,36 @@ spec:
...
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking* -> *Routes* page.
 
-. Select *Edit* from the Actions menu.
+. Click the Actions menu {kebab} next to the Route you want to edit and select *Edit Route*.
```
Comment (bobfuru) [inline modules/deployments-ab-testing-lb.adoc:122]: s/Actions/*Actions*/ ?
Our guidelines mention using asterisks for console items such as menus, although a quick search finds that we are not consistent with this. So whatever is most consistent in this doc book makes sense to me.

File: modules/deployments-ab-testing-lb.adoc [hunk 7]
```diff
@@ -113,23 +113,36 @@ spec:
...
-Traffic weightings are shown on the *Overview* in the expanded rows of the
-applications between which traffic is split.
+. Click *Save*.
+
+[id="deployments-ab-testing-lb-web-new-route_{context}"]
+== Managing weights of an new route using the web console
+
+. Navigate to the *Networking* -> *Routes* page.
+
+. Click the *Create Route* button.
```
Comment (bobfuru) [inline modules/deployments-ab-testing-lb.adoc:133]: Unless this is consistent with other docs in this book, I would shorten as we do with "Save" and write `Click *Create Route*.`

File: modules/deployments-ab-testing-lb.adoc [hunk 8]
```diff
@@ -113,23 +113,36 @@ spec:
...
+
+. Select the *Service*.
+
+. Click *Add Alternate Service*.
+
+. Enter a value for *Weight* and *Alternate Service Weight*. Enter a number between `0` and `255` that depicts relative weight compared with other targets. The default is `100`.
+
+. Select the *Target Port*.
+
+. Click the *Create* button.
```
Comment (bobfuru) [inline modules/deployments-ab-testing-lb.adoc:145]: Click **Create**.
?

**Review by bobfuru — COMMENTED** [review]
A few style guide nits but otherwise LGTM!

**Pull request merged**